.github/workflows/release.yml

```diff
@@ -33,34 +43,76 @@ jobs:
   linux:
     strategy:
       matrix:
-        openssl:
-          - 1_1_0
-          - 1_0_1
-    name: Build - OpenSSL ${{ matrix.openssl }}
+        arch: [amd64, arm64]
+        openssl: [1_0_1, 1_1_1]
+    
+    name: Build - OpenSSL ${{ matrix.openssl }} - Arch ${{ matrix.arch }}
     runs-on: ubuntu-latest
     steps:
+      
```

Comment:
Personally I find these files are much easier to read and grok if there's a newline between each step. I only modified the areas I was working on but I'm happy to either add this to all the jobs or remove it from here before the PR merges.

Reply:
I think updating the areas you touched is reasonable, we can go through the others at some point in the future.